### PR TITLE
API Updates: Identity APIs

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 6f9da2d.
r? @remi-stripe 
cc @stripe/api-libraries

## Changelog
* Add support for Identity VerificationSession and VerificationReport APIs

### CODE REVIEW

File: src/main/java/com/stripe/model/identity/VerificationReport.java [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+    @SerializedName("address")
+    Address address;
+
+    /** Date of birth as it appears in the document. */
+    @SerializedName("dob")
+    DateOfBirth dob;
+
+    /** Details on the verification error. Present when status is {@code unverified}. */
+    @SerializedName("error")
+    DocumentCheckError error;
```
Comment (richardm-stripe) [inline src/main/java/com/stripe/model/identity/VerificationReport.java:148]: Consistent with [`SigmaScheduledQuery.run_error`](https://github.com/stripe/stripe-java/blob/cbe622d4e42676e2d9c5600521f21afc6db45a01/src/main/java/com/stripe/model/sigma/ScheduledQueryRun.java#L31-L32)

File: src/main/java/com/stripe/model/identity/VerificationReport.java [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Document extends StripeObject {
+    /** Address as it appears in the document. */
+    @SerializedName("address")
+    Address address;
+
+    /** Date of birth as it appears in the document. */
+    @SerializedName("dob")
+    DateOfBirth dob;
```
Comment (richardm-stripe) [inline src/main/java/com/stripe/model/identity/VerificationReport.java:144]: Consistent with [Individual.dob](https://github.com/stripe/stripe-java/blob/cbe622d4e42676e2d9c5600521f21afc6db45a01/src/main/java/com/stripe/model/issuing/Cardholder.java#L276-L278)

**Review by remi-stripe — CHANGES_REQUESTED** [review]
sending back to you based on all the things I flagged internally

File: src/main/java/com/stripe/model/identity/VerificationReport.java (hunk 1 above)
Comment (richardm-stripe) [reply src/main/java/com/stripe/model/identity/VerificationReport.java]: image

**Review by remi-stripe — APPROVED** [review]

**Pull request merged**